extensions/opsgenie/README.md

```diff
@@ -1,3 +1,30 @@
 # Opsgenie
 
-The Opsgenie extension can be used to access alerts and incidents from [Opsgenie](https://www.atlassian.com/software/opsgenie). The extension can also used to acknowledge, snooze and close alerts.
+The Opsgenie extension can be used to access alerts and incidents from [Opsgenie](https://www.atlassian.com/software/opsgenie). The extension can also used to acknowledge, unacknowledge, snooze and close alerts.
+
+## 🚀 Getting Started
+You will need the following:
+
+1. API URL of Opsgenie instance - this will depend on where your instance is based-
```

Comment:
Should the two available urls be added here?

- https://api.opsgenie.com
- https://api.eu.opsgenie.com/